docs/tutorial/2-tilt-up.md

```diff
@@ -0,0 +1,143 @@
+
+> We've specified `--port=3366` here to free up the default port in case you want to launch another Tilt project simultaneously.
+> If not overridden, the default Tilt port is `10350`.
+
+You should see output similar to the following in your terminal:
+![Running tilt up in a Terminal window shows "Tilt started on http://localhost:3366/" message](/assets/docimg/tutorial/tilt-up-cli.gif)
+
+Hit `Spacebar` while your terminal is active, and Tilt will launch your default browser.
+(Or navigate to [http://localhost:3366/]() directly in your preferred browser.)
+
+In the next section, we'll explain the Tilt UI. But first, let's dissect what's happening in the background.
+<!-- TODO(milas): this would be a great place for a cheeky graphic about how we're stalling while the builds happen -->
+
+## `Tiltfile`
+When you run `tilt up`, Tilt looks for a special file named `Tiltfile` in the current directory, which defines your dev-environment-as-code.
+
+A `Tiltfile` is written in [Starlark][starlark], a simplified dialect of Python.
+
+> 🐍 Not a Python expert? No worries. Our guides have lots of examples, so you can copy & paste your way to success!
+
+Because your `Tiltfile` is a program, you can configure it with familiar constructs like loops, functions, and arrays.
+This makes the `Tiltfile` more extensible than a configuration file format like JSON or YAML, which requires hard-coding all possible options upfront.
+
+When Tilt executes the `Tiltfile`:
+ 1. Built-in functions like [`k8s_yaml`][api-k8s_yaml] and [`docker_build`][api-docker_build] register information with the Tilt engine
+ 2. Tilt uses the resulting configuration to assemble resources to build and deploy
+ 3. Tilt watches **relevant** source code files so it can trigger an update of the associated resource(s) 
+
+Within Tilt, the `Tiltfile` is itself a resource, so **you can even modify your `Tiltfile` and see the changes without restarting Tilt**!
+
+![Sample Tiltfile code](/assets/docimg/tutorial/tiltfile.png){:class="no-shadow"}
+
+Later on, we'll explore how Tilt makes it possible to optimize this process even more.
+You can skip container re-builds and Pod re-deployments entirely via [Smart Rebuilds with Live Update][tutorial-live-update].
+
+(If you're curious, go ahead and open the [`tilt-avatars` Tiltfile][repo-tilt-avatars-tiltfile] and read through it.
+We won't tell anyone you peeked.)
+
+For now, that's all you need to know!
+<!-- TODO(milas): snarky graphic about how that ^^^ was a galaxy brain info dump? -->
+
+## On Resources
+A "resource" is a bundle of work managed by Tilt. For example: a Docker image to build + a Kubernetes YAML to apply.
+
+> 😶‍🌫️ **Resources don't have to be containers!**
+>
+> Tilt can also [manage locally-executed commands][local-resource] to provide a unified experience no matter how your code runs.  
+
+Resource bundling is **automatic** in most cases: Tilt finds the relationship between bits of work (e.g. `docker_build` + `kubectl apply`).
```

Comment:
@lizzthabet — I made a note of this, since I'm working on the styling of docs!

Reply:
TIL that Firefox doesn't applying bold styling to `<strong>` by default while both Safari/Chrome do!

Reply:
Went ahead and added this to the current design so it's consistent across browsers since it's so tiny: 573dfc1
```css
strong {
  font-weight: bold;
}
```
Safari/Chrome users won't see a difference, while FF users will now know when we are flexing 💪  (get it? strong? ha ha ha)